packages/frontend/src/contexts/ChatContext.tsx

```diff
@@ -171,6 +123,16 @@ export const ChatProvider = ({
 
   unselectChatRef.current = unselectChat
```

Comment:
Although this PR removes some react warnings we should avoid introducing new ones. So either wrap this in a useEffect or add a line // ignore...